x-pack/plugins/observability_solution/uptime/common/constants/client_defaults.ts

```diff
@@ -14,7 +14,7 @@ export const CLIENT_DEFAULTS = {
   /**
    * The application auto refreshes every 60s by default.
    */
-  AUTOREFRESH_INTERVAL: 60 * 1000,
+  AUTOREFRESH_INTERVAL: 5 * 60 * 1000,
```

Comment:
Why is this changing? I don't see it in the PR description. Also, if you do want to change it the comment above is now wrong.

Reply:
i have updated the description !!